README.md

```diff
@@ -1,3 +1,7 @@
+Week 2 - Weekend Challenge
```

Comment:
Good README - clear how to use the application
It would be good to include some more information about how approach, and the design of the application in a diagram.